app/models/card.rb

```diff
@@ -15,9 +16,13 @@ def self.of_user(id)
     where(user_id: id)
   end
 
+  def self.of_deck(id)
+    where(deck_id: id)
+  end
+
   def check_translation(version_of_translation)
     if prepare_word(version_of_translation) == prepare_word(translated_text)
-      self.review_date = Time.now + 5.days
+      self.review_date = Time.now.to_date + 5.days
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.